### PR TITLE
MOSIP-32461, MOSIP-36412 Changing commons  versions for 1.3.x branch

### PR DESCRIPTION
**Pull request by Khuddusshariff0022**

### CODE REVIEW

**Review by ashok-ksharma — CHANGES_REQUESTED** [review]
1. Can we check whether below property is getting used or not: <kernel.object.store.version>1.3.0-SNAPSHOT</kernel.object.store.version>
2. kernel-bom dependency should be changed to the **1.3.0-beta.1**.

File: data-share/pom.xml [hunk 1]
```diff
@@ -27,8 +27,8 @@
 
         <spring-cloud-config.version>2.0.4.RELEASE</spring-cloud-config.version>
         <h2.version>1.4.197</h2.version>
-        <kernel.core.version>1.3.0-SNAPSHOT</kernel.core.version>
-        <kernel.logger.logback.version>1.3.0-SNAPSHOT</kernel.logger.logback.version>
+        <kernel.core.version>1.3.0-beta.1</kernel.core.version>
+        <kernel.logger.logback.version>1.3.0-beta.1</kernel.logger.logback.version>
         <kernel.crypto-jce.version>1.0.6</kernel.crypto-jce.version>
```
Comment (ashok-ksharma) [inline data-share/pom.xml:32]: Are we using this:
<kernel.crypto-jce.version>1.0.6</kernel.crypto-jce.version>

**Review by ashok-ksharma — CHANGES_REQUESTED** [review]

File: data-share/pom.xml (hunk 1 above)
Comment (Khuddusshariff0022) [reply data-share/pom.xml:32]: not using this property. hence removing it.

**Comment by ashok-ksharma** [comment]
@Khuddusshariff0022 If we are not using this then we can remove the same:
 <kernel.object.store.version>1.3.0-SNAPSHOT</kernel.object.store.version>

**Comment by Khuddusshariff0022** [comment]
> @Khuddusshariff0022 If we are not using this then we can remove the same: <kernel.object.store.version>1.3.0-SNAPSHOT</kernel.object.store.version>

not using this property. hence removing it.

**Review by ashok-ksharma — APPROVED** [review]

**Pull request closed without merging**